app/Dao/Post/PostDao.php

```diff
@@ -16,13 +17,40 @@ public function index()
         return Post::latest()->paginate(5);
     }
 
+    //Get Category Data
+    public function create()
+    {
+      return Category::all();
+    }
```

Comment:
**change** 
`public function create() {`
 **to**
`public function getCategories() {`